packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/scanner-api/hardware-scanner-example.ts

```diff
@@ -0,0 +1,41 @@
+import React, {useEffect} from 'react';
+import {
+  Navigator,
+  Screen,
+  Stack,
+  Text,
+  extension,
+} from '@shopify/ui-extensions/point-of-sale';
+
+export default extension('pos.home.modal.render', (root, api) => {
+  api.scanner.scannerDataSubscribable.subscribe((data) => {
+    if (data) {
+      api.cart.addLineItem(12345678, 1);
+    }
```

Comment:
This is sort of weird since the product will very often fail, and then we'll get some sort of error banner, right? I'd just remove this bit.

Reply:
This is outdated, I'll mirror the line to match the react example with `      api.cart.addLineItem(Number(data), 1);`